pages/gallery.html

```diff
@@ -4,9 +4,40 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="../CSS/style.css">
+  <link rel="stylesheet" href="../CSS/gallery.css">
+
   <title>Document</title>
```

Comment:
Remember to update the title for each page! This is what shows up in the browser tab. It should describe the page.